i18n/fr/docusaurus-plugin-content-docs/version-23.04/resources/glossary.md

```diff
@@ -198,6 +200,10 @@ Message avertissant l’utilisateur d’un incident. Il est possible de paramét
 
 Voir [**Moteur de supervision**](#moteur-de-supervision).
 
+## Pack
+
+Un pack contient les informations relatives aux indicateurs qui seront supervisés, leurs seuils d’alerte et les commandes nécessaires pour les contrôles. Le pack apporte une configuration prédéterminée mais celle-ci reste hautement personnalisable. 
```

Comment:
- Remplacer par "relatives aux services" ? (en EN tu dis "related to the services")
- Remplacer par "Le pack fournit une configuration par défaut" ?